src/main/java/groovy/lang/ExpandoMetaClass.java

```diff
@@ -477,10 +477,10 @@ public static void disableGlobally() {
         ExpandoMetaClassCreationHandle.disable();
     }
 
-    /* (non-Javadoc)
-     * @see groovy.lang.MetaClassImpl#initialize()
+    /**
+     * {@inheritDoc}
```

Comment:
This is better for our "api" Javadoc but we currently don't support inheritDoc within Groovydoc:
https://issues.apache.org/jira/browse/GROOVY-3782
so it leaves a bit of noise in the "gapi" documentation - still, not a huge issue I guess.

Reply:
But this is a java file... do you mean javadocs in java files are also processed by `groovydoc`?

Reply:
Yes, the gapi (Groovydoc ) is currently the combined documentation produced by converting java -> groovy and then running groovydoc on all. But as I said, not a big deal for this PR. It was more to inform why it was the way it was.

Reply:
Thanks for the clarification.